test_runner/docs/ascii/flank.jar_-ios-run.adoc

```diff
@@ -146,15 +146,15 @@ Configuration is read from flank.yml
 *--disable-results-upload*::
   Disables flank results upload on gcloud storage.
 
-*--test*=_<test>_::
-  The path to the test package (a zip file containing the iOS app and XCTest files). The given path may be in the local filesystem or in Google Cloud Storage using a URL beginning with gs://. Note: any .xctestrun file in this zip file will be ignored if --xctestrun-file is specified.
-
 *--xctestrun-file*=_<xctestrunFile>_::
   The path to an .xctestrun file that will override any .xctestrun file contained in the --test package. Because the .xctestrun file contains environment variables along with test methods to run and/or ignore, this can be useful for customizing or sharding test suites. The given path may be in the local filesystem or in Google Cloud Storage using a URL beginning with gs://.
 
 *--xcode-version*=_<xcodeVersion>_::
   The version of Xcode that should be used to run an XCTest. Defaults to the latest Xcode version supported in Firebase Test Lab. This Xcode version must be supported by all iOS versions selected in the test matrix.
 
+*--test*=_<test>_::
+  The path to the test package (a zip file containing the iOS app and XCTest files). The given path may be in the local filesystem or in Google Cloud Storage using a URL beginning with gs://. Note: any .xctestrun file in this zip file will be ignored if --xctestrun-file is specified.
```

Comment:
Just to make sure, all theses small changes are from the auto doc generation?

Reply:
yeah, they generate different output for same data 😞

Reply:
Yes, on every build, we have generated docs with changes like that.

Reply:
I think we should disable the doc generation by default until this is fixed.

@Sloox do you want to handle this contribution?

I think you just have to comment out

https://github.com/Flank/flank/blob/5ea15cf881b1022c74861e618c59e8ac63386501/test_runner/build.gradle.kts#L284-L286

Reply:
@bootstraponline Sure. is there an issue surrounding this so i can tag the branch correctly? or is a simple "Disable doc generation" sufficent?

Reply:
this is small enough that we don't need an issue 🙂 "Disable doc generation" sounds great!